modules/test/playwright/fixtures/featureFlagsTest.ts

```diff
@@ -22,6 +33,30 @@ export interface FeatureFlags {
 
 const test = mergeTests(backendPageTest);
 
+const getCompanyId = (value: boolean | ScopedFeatureFlagValue) => {
```

Comment:
I guess this method could return the current company id in case it's not a system scoped feature flag. But I can't call `Liferay.ThemeDisplay.getCompanyId()` in here. @ethib137 @ilzamcmed does you guys know why?

![image](https://github.com/user-attachments/assets/8f8e7343-7a16-473a-9d09-52d0a1befa5a)

We could simplify the method to 

```typescript
const getCompanyId = (value: boolean | ScopedFeatureFlagValue) => {
	if (typeof value !== 'boolean') {
		const scopedFeatureFlagValue = value as ScopedFeatureFlagValue;

		if (scopedFeatureFlagValue.system) {
			return 0;
		}
	}

	return Number(Liferay.ThemeDisplay.getCompanyId());
};

```